react/features/facial-recognition/constants.js

```diff
@@ -9,3 +9,18 @@ export const FACIAL_EXPRESSION_EMOJIS = {
     fearful: '😨',
     disgusted: '🤢'
 };
+
+/**
+ * Time used for detection interval when facial expressions worker uses webgl backend.
+ */
+export const WEBGL_TIME_INTERVAL = 1000;
+
+/**
+ * Time used for detection interval when facial expression worker uses cpu backend.
+ */
+export const CPU_TIME_INTERVAL = 6000;
+
+/**
+ * Time is ms used for sending expression.
+ */
+export const WEBHOOK_SEND_TIME_INTERVAL = 15000;
```

Comment:
i don't think this belongs in client code @saghul @damencho wdyt?

Reply:
The client needs to have defaults. If you want to control it, add a config option, but still leave a default value.